docs-sidebar.js

```diff
@@ -180,7 +180,18 @@ const sidebars = {
             'accounts/reconciliation',
             'transactions/payees',
             'transactions/bulk-editing',
-            'advanced/bank-sync',
+            {
+              type: 'category',
+              label: 'Connecting Your Bank',
+              link: {
+                type: 'doc',
+                id: 'advanced/bank-sync',
+                    },
+                    items: [
+                    'advanced/bank-sync/gocardless',
+                    'advanced/bank-sync/simplefin',
+                    ],
+            },
```

Comment:
The formatting here is a little wonky

```suggestion
            {
              type: 'category',
              label: 'Connecting Your Bank',
              link: {
                type: 'doc',
                id: 'advanced/bank-sync',
              },
              items: [
                'advanced/bank-sync/gocardless',
                'advanced/bank-sync/simplefin',
              ],
            },
```